### PR TITLE
Issue 38736: Update to PK calculations

### PR DESCRIPTION
**Pull request by labkey-jeckels**
Still pending OK from requesting customer on some details, but getting the tests and review underway.

### CODE REVIEW

File: src/org/labkey/targetedms/view/pharmacokinetics.jsp [hunk 1]
```diff
@@ -340,7 +342,8 @@
             if(!value){
                 return '';
             }
-            return LABKEY.Utils.roundNumber(value,3).toLocaleString(undefined, {minimumFractionDigits: 3,maximumFractionDigits: 3});
+            return Number.parseFloat(value).toPrecision(4);
+            return LABKEY.Utils.roundNumber(value).toLocaleString(undefined, {maximumSignificantDigits: 4});
```
Comment (labkey-ankurj) [inline src/org/labkey/targetedms/view/pharmacokinetics.jsp]: I guess the old return statement didn't get removed.

Comment (labkey-ankurj) [inline src/org/labkey/targetedms/view/pharmacokinetics.jsp]: two return statements left here.

Comment (labkey-jeckels) [reply src/org/labkey/targetedms/view/pharmacokinetics.jsp]: Thanks, now removed!

**Comment by labkey-jeckels** [comment]
Closing this one, have a new one against the 19.3 branch.

**Pull request closed without merging**